orchestration/templates/ingest-hca.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-hca
+spec:
+  entrypoint: main
+  serviceAccountName: {{ .Values.serviceAccount.k8sName }}
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: source-bucket-name
+          - name: staging-bucket-name
+
+      dag:
+        tasks:
+          # Pre-process the data found in a source bucket
+          - name: pre-process-source
+            templateRef:
+              name: run-dataflow
+              template: main
```

Comment:
Since it's in the same file, it should just be:
```
template: run-dataflow
```
instead of the `templateRef` block